package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "radicle-avatar",
+  "version": "0.1.0",
+  "author": "radicle team <dev@radicle.xyz>",
+  "license": "MIT OR Apache-2.0",
+  "repository": "git@github.com:radicle-dev/radicle-avatar.git",
+  "main": "./dist/js/index.js",
+  "types": "./dist/js/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "test": "yarn regenerate-fixtures && jest",
+    "clean": "rm -rf ./dist",
+    "regenerate-fixtures": "rm -rf ./fixtures.json && cargo run --example generate-fixtures",
+    "build": "yarn run clean && tsc --project tsconfig.package.json",
+    "prepare": "yarn run build"
```

Comment:
I think this script should be called `prepack`. `prepare` is not a [Yarn lifecycle script](https://yarnpkg.com/advanced/lifecycle-scripts). The `prepack` script is also run when the package is installed as a git dependency. See also the [NPM script docs](https://docs.npmjs.com/cli/v7/using-npm/scripts)

```suggestion
    "prepack": "yarn run build"
```